src/redux/books/books.js

```diff
@@ -1,25 +1,15 @@
-import { v4 as uuidv4 } from 'uuid';
+// import { v4 as uuidv4 } from 'uuid';
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const LOAD_API = 'bookstore/books/LOAD_API';
+// const ADD_BOOK_TO_API = 'ADD_BOOK_TO_API';
+// const ADD_BOOK_TO_API = 'bookstore/books/ADD_BOOK_TO_API'
```

Comment:
- [ ] Old commented code makes your code messy, kindly try to always remove unnecessary commented code from your code before pushing it to Github

Reply:
@Ismailco Thank you for pointing that out. It always skips my mind😂.